utils/views.py

```diff
@@ -54,31 +86,14 @@ async def interaction_check(self: Self, interaction: discord.Interaction) -> boo
 
         return True
 
-    @discord.ui.select(  # type: ignore
-        custom_id="select_cog",
-        placeholder="Choose a category",
-        min_values=1,
-        max_values=1,
-        row=1,
-    )
-    async def cog_select(self: Self, interaction: discord.Interaction, select: discord.ui.Select[Any]) -> None:
-        if select.values[0] == "Close":
-            if not interaction.message:
-                return
-
-            return await interaction.message.delete()
-
-        cog: commands.Cog = interaction.client.get_cog(select.values[0])  # type: ignore
-        command_list: str = ""
-        for command in cog.get_commands():
-            command_list += f"{command.name}\n"
-
-        embed: discord.Embed = discord.Embed(title=cog.qualified_name, description=command_list)
-        await interaction.response.edit_message(embed=embed)
-
 
 class InviteView(discord.ui.View):
     def __init__(self: Self, ctx: AloneContext) -> None:
-        self.ctx: AloneContext = ctx
         super().__init__(timeout=None)
-        self.add_item(discord.ui.Button(label="Invite", url=discord.utils.oauth_url(self.ctx.bot.user.id)))
+
+        user = ctx.bot.user
```

Comment:
can you explain your thought process here? why did you use ctx.bot here and defined self.ctx after, even tho you didn't need it?